Gemfile

```diff
@@ -20,6 +20,7 @@ gem "gds-sso", "12.1.0"
 gem 'bunny', '2.5.1'
 gem 'whenever', '0.9.4', require: false
 gem "govuk_sidekiq", "~> 0.0"
+gem 'sidekiq', git: "https://github.com/alphagov/sidekiq.git", branch: "fix-testing-with-overridden-queue"
```

Comment:
It could go in govuk_sidekiq, but that would still require changes here, so for simplicity, I just made the change here.

As for upgrading sidekiq, personally, I prefer dependency changes to be kept separate from feature changes, unless they are actually related.